examples/index.html

```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
-<head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<html lang="en-US"><head>
+  <meta charset="UTF-8">
```

Comment:
I think in the other files the lowercase "utf-8" is used, but that it doesn't make a functional difference

Reply:
Right. I don't care about casing here...